### PR TITLE
allow custom broadcast mode

### PR DESCRIPTION
**Pull request by mmsqe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - [#86](https://github.com/crypto-com/pystarport/pull/86) support golang relayer
 - [#91](https://github.com/crypto-com/pystarport/pull/91) replace block with sync broadcast mode
 - [#93](https://github.com/crypto-com/pystarport/pull/93) add default gas prices for golang relayer
+- []() allow custom broadcast mode
```
Comment (mmsqe) [inline CHANGELOG.md:8]:
```suggestion
- [#94](https://github.com/crypto-com/pystarport/pull/94) allow custom broadcast mode
```

File: pystarport/cluster.py [hunk 2]
```diff
@@ -214,6 +214,7 @@ def create_node(
         # init home directory
         self.init(i)
         home = self.home(i)
+        mode = self.config.get("broadcast-mode", "sync")
```
Comment (yihuang) [inline pystarport/cluster.py:217]: I guess we can add a `client-config` field.

Comment (yihuang) [reply pystarport/cluster.py:217]: I think there's one already: `client_config`.

Comment (mmsqe) [reply pystarport/cluster.py]: seems we need allow custom when [create_node](https://github.com/crypto-org-chain/chain-main/blob/master/integration_tests/test_staking.py#L135)

File: pystarport/cluster.py [hunk 3]
```diff
@@ -214,6 +215,7 @@ def create_node(
         # init home directory
         self.init(i)
         home = self.home(i)
+        mode = self.config.get("broadcast-mode", broadcastmode)
```
Comment (yihuang) [inline pystarport/cluster.py:218]: remove this? `config` is supposed to be `config.toml`.

Comment (yihuang) [inline pystarport/cluster.py:218]: don't need this?

Comment (mmsqe) [reply pystarport/cluster.py]: yea, reverted

**Review by yihuang — APPROVED** [review]

**Pull request merged**